src/View.php

```diff
@@ -76,6 +77,11 @@ class View implements DynamicContentAwareInterface
      */
     protected Theme $theme;
 
+    /**
+     * @var FragmentCache
```

Comment:
PhpDoc and prperty type are different. Is it ok?

Reply:
Thank. Fixed.